src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -718,33 +718,35 @@ void BaseCouplingScheme::advanceTXTWriters()
       if (convMeasure.level > 0)
         hasCoarseModelOptimization = true;
 
-    _iterationsWriter->writeData("Timesteps", _timesteps - 1);
-    _iterationsWriter->writeData("Total_Iterations", _totalIterations);
+    _iterationsWriter->writeData("Timestep", _timesteps - 1);
+    _iterationsWriter->writeData("TotalIterations", _totalIterations);
     _iterationsWriter->writeData("Iterations", _iterations);
     if (hasCoarseModelOptimization) {
-      _iterationsWriter->writeData("Total_Iterations_Surrogate_Model", _totalIterationsCoarseOptimization);
-      _iterationsWriter->writeData("Iterations_Surrogate_Model", _iterationsCoarseOptimization);
+      _iterationsWriter->writeData("TotalIterations_SurrogateModel", _totalIterationsCoarseOptimization);
+      _iterationsWriter->writeData("Iterations_SurrogateModel", _iterationsCoarseOptimization);
     }
     int converged = _iterations < _maxIterations ? 1 : 0;
     _iterationsWriter->writeData("Convergence", converged);
 
+    int i = -1;
```

Comment:
We don't need `i` anywhere, right?

Reply:
`i` is used in calculation of the residual below (line742)

Reply:
Then it's ok, I guess! :-)